demo/stt_or_tts/run.py

```diff
@@ -10,13 +10,12 @@
     title=None,
     examples=tts_examples,
     description="Give me something to say!",
-    cache_examples=False
 )
 
 stt_demo = gr.load(
     "huggingface/facebook/wav2vec2-base-960h",
     title=None,
-    inputs="mic",
+    inputs=gr.Microphone(type="filepath"),
```

Comment:
The input component needs to have `type="filepath"` to preprocess the values correctly. Not sure how/if it was working before